.github/workflows/codespell.yml

```diff
@@ -0,0 +1,18 @@
+on:
+    push:
+        branches: [master]
+    pull_request:
+
+name: Spell Check
+ 
+jobs:
+    codespell:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: codespell-project/actions-codespell@master
+              with:
+                  check_filenames: true
+                  only_warn: 1
+                  ignore_words_list: cas,objext,ba,probleme,formend,compiletime,chage,hve,files,ans,inout,testin,files\',creat
+                  skip: .idea,.git,*.png,*.zip,*.gif,*.jpg,*.log,*.sub,ChangeLog,uploadcodecov.sh,codespell.yml,tmp,vendor,cache,bundles,ace,*d3.min*,bootstrap*,FileSaver*,jquery*,jscolor*,config.guess
```

Comment:
The `skip` config seems to allow directory paths, so why not `./lib/vendor`? Similar for some other file/directory names.